src/app/index.tsx

```diff
@@ -1,16 +1,38 @@
+import { Session } from '@supabase/supabase-js';
 import { Link } from 'expo-router';
+import { useEffect, useState } from 'react';
 import { Text, View } from 'react-native';
 import { TouchableOpacity } from 'react-native-gesture-handler';
+import 'react-native-url-polyfill/auto';
+import DummyQueries from '../../lib/DummyQueries';
+import { supabase } from '../../lib/supabase';
 import styles from './styles';
 
 function StartScreen() {
+  const [session, setSession] = useState<Session | null>(null);
```

Comment:
eslint is complaining about `eslint[@typescript-eslint/no-shadow](https://typescript-eslint.io/rules/no-shadow)` and we can resolve this by renaming the state variable `session`.

Reply:
Also eslint will complain if we do not use this variable so for now, we can either remove it or get the session user in order to use the variable.